akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -495,7 +494,7 @@ object Attributes {
    */
   def name(name: String): Attributes =
     if (name == null || name.isEmpty) none
-    else Attributes(Name(URLEncoder.encode(name, ByteString.UTF_8)))
+    else Attributes(Name(name))
```

Comment:
@pyoio do you know if this will be ok in Lightbend Telemetry? Anything that need adjustments in Grafana dashboards or similar?

Reply:
Many tests failed. I didn't look carefully into the issue due to some other business. Hopefully I will look into that and lightbend telemetry tomorrow.